tests/frontier/identity_precompile/common.py

```diff
@@ -0,0 +1,93 @@
+"""Common utilities for the Identity Precompile tests."""
+
+from dataclasses import dataclass
+from typing import Tuple
+
+from ethereum_test_base_types.composite_types import Storage
+from ethereum_test_tools import (
+    Bytecode,
+)
+from ethereum_test_tools import Opcodes as Op
+
+
+@dataclass(frozen=True)
+class Constants:
+    """Constants for the Identity Precompile."""
+
+    IDENTITY_PRECOMPILE_ADDRESS = 0x04
+
+
+class CallArgs:
+    """Defines inputs to CALL for the Identity precompile."""
+
+    def __init__(
+        self,
+        gas=0x1F4,
+        value=0x0,
+        args_offset=0x0,
+        args_size=0x20,
+        ret_offset=0x0,
+        ret_size=0x20,
+    ):
+        """Create a new instance with the provided values."""
+        self.gas = gas
+        self.value = value
+        self.args_offset = args_offset
+        self.args_size = args_size
+        self.ret_offset = ret_offset
+        self.ret_size = ret_size
```

Comment:
```suggestion
class CallArgs(BaseModel):
    """Defines inputs to CALL for the Identity precompile."""

    gas: int = 0x1F4
    address: int = Constants.IDENTITY_PRECOMPILE_ADDRESS
    value: int = 0x0
    args_offset: int = 0x0
    args_size: int = 0x20
    ret_offset: int = 0x0
    ret_size: int = 0x20
```

Pydantic would be helpful here, and also can be used to pass in the parameters more nicely to the opcode (see next comment).